src/FluentValidation/IValidationContext.cs

```diff
@@ -238,14 +238,13 @@ public ValidationContext(T instanceToValidate, PropertyChain propertyChain, IVal
 		/// Creates a new validation context for use with a child validator
 		/// </summary>
 		/// <param name="instanceToValidate"></param>
-		/// <param name="preserveParentContext"></param>
 		/// <param name="selector"></param>
 		/// <returns></returns>
-		public ValidationContext<TChild> CloneForChildValidator<TChild>(TChild instanceToValidate, bool preserveParentContext = false, IValidatorSelector selector = null) {

```

Comment:
This is a breaking change to the public API. The caller should be updated to pass `true` rather than changing this method.